testsuite/tests/typing-safe-linking/Makefile

```diff
@@ -3,6 +3,13 @@
 SOURCES = a.ml b_bad.ml
 OBJECTS = $(SOURCES:%.ml=%.cmo)
 
+default:
+	@if $(SAFE_STRING) ; then \
+	  echo " ... testing 'b_bad.ml' => skipped" ; \
```

Comment:
You could remove the definition of `A.y` and the last line of `b_bad.ml` because the program is never executed anyway, then you could compile in `-safe-string` mode. What I don't understand is that I get a warning 8 with or without the `-safe-string` option. @garrigue ?